modules/ROOT/pages/3.7_XML_examples_and_tools.adoc

```diff
@@ -3,7 +3,7 @@ include::partial$attributes.adoc[]
 
 The fact of presenting the data structures as a spread-sheet book had an additional reason: *to use the spread-sheet as an elementary prototype tool to generate the XML instances of the criteria for the ESPD Request and ESDP Response documents*.
 
-Thus the folder link:{url-tree}/xml-examples/__xslt__/ods-data-structures-to-ESPD-xml[xml-examples/__xslt__/ods-data-structures-to-ESPD-xml] contains four XSL style-sheets that facilitate the generation of the complete set of criteria required in an ESDP Request or in an ESDP Response XML file.
+Thus the folder link:{url-tree-tmp}/xml-examples/__xslt__/ods-data-structures-to-ESPD-xml[xml-examples/__xslt__/ods-data-structures-to-ESPD-xml] contains four XSL style-sheets that facilitate the generation of the complete set of criteria required in an ESDP Request or in an ESDP Response XML file.
```

Comment:
link:{url-tree-tmp}/xml-examples/__xslt__/ods-data-structures-to-ESPD-xml[xml-examples/__xslt__/ods-data-structures-to-ESPD-xml] does not work

Reply:
Fixed for another comment.